style.css

```diff
@@ -145,30 +152,31 @@ hr{
 
 
 /* box3 */
-.nav-search{
+.nav-search {
     display: flex;
     justify-content: space-evenly;
     background-color: #fedb68;
     width: 620px;
     height: 40px;
     border-radius: 4px;
 }
-.search-select{
+
+.search-select {
     background-color: #f3f3f3;
     width: 50px;
     text-align: center;
     border-top-left-radius: 4px;
     border-bottom-left-radius: 4px;
-    border: none ;
+    border: none;
 }
 
-.search-input{
+.search-input {

```

Comment:
you have just added and removed spaces. Could you explain why?

Reply:
formatting issue maybe,